images/base/files/etc/containerd/config.toml

```diff
@@ -40,3 +40,6 @@ version = 2
   tolerate_missing_hugepages_controller = true
   # restrict_oom_score_adj needs to be true when running inside UserNS (rootless)
   restrict_oom_score_adj = false
+  # enable CDI https://github.com/cncf-tags/container-device-interface
+  enable_cdi = true
```

Comment:
why is this default false? 👀

Reply:
@estesp @mikebrow can you help us here ? ^^

Reply:
I see, is a 2.0 thing https://github.com/containerd/containerd/commit/c8e8a093ce3dd9231173042e9828a679972f5d6b

Reply:
correct